2020-05-29 11:21:59,083 WARN  [org.hibernate.engine.jdbc.internal.JdbcServicesImpl] (main) HHH000342: Could not obtain connection to query metadata : Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:22:03,367 INFO  [com.nec.asia.nic.framework.security.ad.impl.AuthenticationServiceTestCase] (pool-3-thread-2) start authenticationService - authenticate
2020-05-29 11:22:24,563 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-3-thread-2) SQL Error: 0, SQLState: null
2020-05-29 11:22:24,564 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-3-thread-2) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:22:24,590 INFO  [com.nec.asia.nic.framework.security.ad.impl.AuthenticationServiceTestCase] (pool-3-thread-2)  end  authenticationService - authenticate
2020-05-29 11:22:24,599 INFO  [com.nec.asia.nic.util.FontTest] (pool-3-thread-2) Word: I'm trying to write application which need to draw many strings using Java Graphics class in Java.
2020-05-29 11:22:24,842 INFO  [com.nec.asia.nic.util.FontTest] (pool-3-thread-2) Font: Verdana
length:98	width:612
2020-05-29 11:22:24,938 INFO  [com.nec.asia.nic.util.FontTest] (pool-3-thread-2) Font: Haettenschweiler
length:98	width:351
2020-05-29 11:22:24,950 INFO  [com.nec.asia.nic.util.FontTest] (pool-3-thread-2) Font: Century Gothic
length:98	width:572
2020-05-29 11:22:25,436 WARN  [org.hibernate.engine.jdbc.internal.JdbcServicesImpl] (pool-1-thread-2) HHH000342: Could not obtain connection to query metadata : Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:22:26,924 INFO  [com.nec.asia.nic.framework.admin.code.dao.SiteRepositoryDaoTestCase] (pool-1-thread-6) [findBySiteId]
2020-05-29 11:22:26,924 INFO  [com.nec.asia.nic.framework.admin.rbac.dao.FunctionsDaoTestCase] (pool-1-thread-1) start Functions dao - save
2020-05-29 11:22:26,928 INFO  [com.nec.asia.nic.framework.admin.code.service.CodesServiceTestCase] (pool-1-thread-7) start CodesService - delete
2020-05-29 11:22:26,943 INFO  [com.nec.asia.nic.framework.admin.code.dao.ParametersDaoTestCase] (pool-1-thread-4) start Parameters dao - findParametersByScope
2020-05-29 11:22:26,948 INFO  [com.nec.asia.nic.framework.admin.code.service.ParametersServiceTestCase] (pool-1-thread-8) start ParametersService - findAllForPaginationByScope
2020-05-29 11:22:26,951 INFO  [com.nec.asia.nic.framework.admin.code.dao.PaymentDefDaoTestCase] (pool-1-thread-5) start PaymentDef dao - findPaymentDefByTransType
2020-05-29 11:22:26,971 INFO  [com.nec.asia.nic.framework.admin.code.dao.CodeDaoTestCase] (pool-1-thread-2) start Codes dao - save
2020-05-29 11:22:26,977 INFO  [com.nec.asia.nic.framework.admin.code.dao.CodeValueDaoTestCase] (pool-1-thread-3) start dao - findAllByCodeId
2020-05-29 11:22:47,975 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-2) SQL Error: 0, SQLState: null
2020-05-29 11:22:47,976 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-2) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:22:47,979 ERROR [com.nec.asia.nic.framework.admin.code.dao.CodeDaoTestCase] (pool-1-thread-2) error in Save: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Could not open connection
2020-05-29 11:22:47,984 INFO  [com.nec.asia.nic.framework.admin.code.dao.CodeDaoTestCase] (pool-1-thread-2) start Codes dao - update
2020-05-29 11:23:08,979 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-8) SQL Error: 0, SQLState: null
2020-05-29 11:23:08,981 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-8) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:23:08,989 ERROR [com.nec.asia.nic.framework.admin.code.service.impl.ParametersServiceImpl] (pool-1-thread-8) Error occurred while getting the parameters. Exception: Could not open connection
org.hibernate.exception.GenericJDBCException: Could not open connection
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:54)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.LogicalConnectionImpl.obtainConnection(LogicalConnectionImpl.java:235)
	at org.hibernate.engine.jdbc.internal.LogicalConnectionImpl.getConnection(LogicalConnectionImpl.java:171)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.connection(StatementPreparerImpl.java:63)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$5.doPrepare(StatementPreparerImpl.java:162)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:186)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareQueryStatement(StatementPreparerImpl.java:160)
	at org.hibernate.loader.Loader.prepareQueryStatement(Loader.java:1885)
	at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:1862)
	at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:1839)
	at org.hibernate.loader.Loader.doQuery(Loader.java:910)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:355)
	at org.hibernate.loader.Loader.doList(Loader.java:2554)
	at org.hibernate.loader.Loader.doList(Loader.java:2540)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2370)
	at org.hibernate.loader.Loader.list(Loader.java:2365)
	at org.hibernate.loader.criteria.CriteriaLoader.list(CriteriaLoader.java:126)
	at org.hibernate.internal.SessionImpl.list(SessionImpl.java:1718)
	at org.hibernate.internal.CriteriaImpl.list(CriteriaImpl.java:380)
	at org.hibernate.internal.CriteriaImpl.uniqueResult(CriteriaImpl.java:402)
	at com.nec.asia.nic.framework.dao.impl.GenericHibernateDao.findAllForPagination(GenericHibernateDao.java:280)
	at com.nec.asia.nic.framework.admin.code.dao.impl.ParametersDaoImpl.findAllForPaginationByScope(ParametersDaoImpl.java:43)
	at com.nec.asia.nic.framework.admin.code.dao.impl.ParametersDaoImpl$$FastClassBySpringCGLIB$$15416b6d.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:720)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:655)
	at com.nec.asia.nic.framework.admin.code.dao.impl.ParametersDaoImpl$$EnhancerBySpringCGLIB$$4afeaadc.findAllForPaginationByScope(<generated>)
	at com.nec.asia.nic.framework.admin.code.service.impl.ParametersServiceImpl.findAllForPaginationByScope(ParametersServiceImpl.java:51)
	at com.nec.asia.nic.framework.admin.code.service.impl.ParametersServiceImpl$$FastClassBySpringCGLIB$$b9c42b0d.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:720)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:655)
	at com.nec.asia.nic.framework.admin.code.service.impl.ParametersServiceImpl$$EnhancerBySpringCGLIB$$9592167c.findAllForPaginationByScope(<generated>)
	at com.nec.asia.nic.framework.admin.code.service.ParametersServiceTestCase.testFindAllForPaginationByScope(ParametersServiceTestCase.java:120)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:254)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:89)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:387)
	at org.apache.maven.surefire.junitcore.pc.InvokerStrategy.schedule(InvokerStrategy.java:54)
	at org.apache.maven.surefire.junitcore.pc.Scheduler.schedule(Scheduler.java:346)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:193)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:387)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.apache.commons.dbcp.SQLNestedException: Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1549)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at org.hibernate.engine.jdbc.connections.internal.DatasourceConnectionProviderImpl.getConnection(DatasourceConnectionProviderImpl.java:139)
	at org.hibernate.internal.AbstractSessionImpl$NonContextualJdbcConnectionAccess.obtainConnection(AbstractSessionImpl.java:380)
	at org.hibernate.engine.jdbc.internal.LogicalConnectionImpl.obtainConnection(LogicalConnectionImpl.java:228)
	... 73 more
Caused by: java.sql.SQLRecoverableException: IO Error: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:458)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:546)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:236)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:32)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:521)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	... 78 more
Caused by: oracle.net.ns.NetException: The Network Adapter could not establish the connection
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:392)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:434)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:687)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:247)
	at oracle.jdbc.driver.T4CConnection.connect(T4CConnection.java:1102)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:320)
	... 86 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:150)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:133)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:370)
	... 91 more
2020-05-29 11:23:08,996 ERROR [ERRORS] (pool-1-thread-8) CoreException. Cause: Could not open connection
com.nec.asia.nic.framework.admin.code.exception.ParametersServiceException: CoreException. Cause: Could not open connection
	at com.nec.asia.nic.framework.admin.code.service.impl.ParametersServiceImpl.findAllForPaginationByScope(ParametersServiceImpl.java:54)
	at com.nec.asia.nic.framework.admin.code.service.impl.ParametersServiceImpl$$FastClassBySpringCGLIB$$b9c42b0d.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:720)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:655)
	at com.nec.asia.nic.framework.admin.code.service.impl.ParametersServiceImpl$$EnhancerBySpringCGLIB$$9592167c.findAllForPaginationByScope(<generated>)
	at com.nec.asia.nic.framework.admin.code.service.ParametersServiceTestCase.testFindAllForPaginationByScope(ParametersServiceTestCase.java:120)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:254)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:89)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:387)
	at org.apache.maven.surefire.junitcore.pc.InvokerStrategy.schedule(InvokerStrategy.java:54)
	at org.apache.maven.surefire.junitcore.pc.Scheduler.schedule(Scheduler.java:346)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:193)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:387)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.hibernate.exception.GenericJDBCException: Could not open connection
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:54)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.LogicalConnectionImpl.obtainConnection(LogicalConnectionImpl.java:235)
	at org.hibernate.engine.jdbc.internal.LogicalConnectionImpl.getConnection(LogicalConnectionImpl.java:171)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.connection(StatementPreparerImpl.java:63)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$5.doPrepare(StatementPreparerImpl.java:162)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:186)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareQueryStatement(StatementPreparerImpl.java:160)
	at org.hibernate.loader.Loader.prepareQueryStatement(Loader.java:1885)
	at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:1862)
	at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:1839)
	at org.hibernate.loader.Loader.doQuery(Loader.java:910)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:355)
	at org.hibernate.loader.Loader.doList(Loader.java:2554)
	at org.hibernate.loader.Loader.doList(Loader.java:2540)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2370)
	at org.hibernate.loader.Loader.list(Loader.java:2365)
	at org.hibernate.loader.criteria.CriteriaLoader.list(CriteriaLoader.java:126)
	at org.hibernate.internal.SessionImpl.list(SessionImpl.java:1718)
	at org.hibernate.internal.CriteriaImpl.list(CriteriaImpl.java:380)
	at org.hibernate.internal.CriteriaImpl.uniqueResult(CriteriaImpl.java:402)
	at com.nec.asia.nic.framework.dao.impl.GenericHibernateDao.findAllForPagination(GenericHibernateDao.java:280)
	at com.nec.asia.nic.framework.admin.code.dao.impl.ParametersDaoImpl.findAllForPaginationByScope(ParametersDaoImpl.java:43)
	at com.nec.asia.nic.framework.admin.code.dao.impl.ParametersDaoImpl$$FastClassBySpringCGLIB$$15416b6d.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:720)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:655)
	at com.nec.asia.nic.framework.admin.code.dao.impl.ParametersDaoImpl$$EnhancerBySpringCGLIB$$4afeaadc.findAllForPaginationByScope(<generated>)
	at com.nec.asia.nic.framework.admin.code.service.impl.ParametersServiceImpl.findAllForPaginationByScope(ParametersServiceImpl.java:51)
	... 44 more
Caused by: org.apache.commons.dbcp.SQLNestedException: Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1549)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at org.hibernate.engine.jdbc.connections.internal.DatasourceConnectionProviderImpl.getConnection(DatasourceConnectionProviderImpl.java:139)
	at org.hibernate.internal.AbstractSessionImpl$NonContextualJdbcConnectionAccess.obtainConnection(AbstractSessionImpl.java:380)
	at org.hibernate.engine.jdbc.internal.LogicalConnectionImpl.obtainConnection(LogicalConnectionImpl.java:228)
	... 73 more
Caused by: java.sql.SQLRecoverableException: IO Error: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:458)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:546)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:236)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:32)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:521)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	... 78 more
Caused by: oracle.net.ns.NetException: The Network Adapter could not establish the connection
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:392)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:434)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:687)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:247)
	at oracle.jdbc.driver.T4CConnection.connect(T4CConnection.java:1102)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:320)
	... 86 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:150)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:133)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:370)
	... 91 more
2020-05-29 11:23:09,000 INFO  [com.nec.asia.nic.framework.admin.code.service.ParametersServiceTestCase] (pool-1-thread-8) start ParametersService - Save
2020-05-29 11:23:29,997 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-7) SQL Error: 0, SQLState: null
2020-05-29 11:23:30,000 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-7) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:23:30,022 INFO  [com.nec.asia.nic.framework.admin.code.service.CodesServiceTestCase] (pool-1-thread-7) start CodesService - Save
2020-05-29 11:23:51,018 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-5) SQL Error: 0, SQLState: null
2020-05-29 11:23:51,020 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-5) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:23:51,036 INFO  [com.nec.asia.nic.framework.admin.code.dao.PaymentDefDaoTestCase] (pool-1-thread-5) start PaymentDef dao - save
2020-05-29 11:24:12,028 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-3) SQL Error: 0, SQLState: null
2020-05-29 11:24:12,030 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-3) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:24:12,036 INFO  [com.nec.asia.nic.framework.admin.code.dao.CodeValueDaoTestCase] (pool-1-thread-3) start CodeValue dao - findAllForPagination
2020-05-29 11:24:12,041 INFO  [com.nec.asia.nic.framework.admin.code.dao.CodeValueDaoTestCase] (pool-1-thread-3) Testing data loading for page 1.
2020-05-29 11:24:33,035 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-4) SQL Error: 0, SQLState: null
2020-05-29 11:24:33,037 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-4) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:24:33,050 INFO  [com.nec.asia.nic.framework.admin.code.dao.ParametersDaoTestCase] (pool-1-thread-4) start Parameters dao - save
2020-05-29 11:24:54,054 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-6) SQL Error: 0, SQLState: null
2020-05-29 11:24:54,055 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-6) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:24:54,058 INFO  [com.nec.asia.nic.framework.admin.code.dao.SiteRepositoryDaoTestCase] (pool-1-thread-6) start SiteRepository dao - save
2020-05-29 11:25:15,058 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-1) SQL Error: 0, SQLState: null
2020-05-29 11:25:15,059 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-1) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:25:15,063 INFO  [com.nec.asia.nic.framework.admin.rbac.dao.FunctionsDaoTestCase] (pool-1-thread-1) start Functions dao - update
2020-05-29 11:25:36,062 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-6) SQL Error: 0, SQLState: null
2020-05-29 11:25:36,063 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-6) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:25:36,067 INFO  [com.nec.asia.nic.framework.admin.code.dao.SiteRepositoryDaoTestCase] (pool-1-thread-6) start SiteRepository dao - update
2020-05-29 11:25:57,068 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-4) SQL Error: 0, SQLState: null
2020-05-29 11:25:57,070 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-4) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:25:57,077 INFO  [com.nec.asia.nic.framework.admin.code.dao.ParametersDaoTestCase] (pool-1-thread-4) start Parameters dabean. update
2020-05-29 11:26:18,075 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-3) SQL Error: 0, SQLState: null
2020-05-29 11:26:18,076 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-3) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:26:18,080 INFO  [com.nec.asia.nic.framework.admin.code.dao.CodeValueDaoTestCase] (pool-1-thread-3) start CodeValue dao - findAllByExampleForPagination
2020-05-29 11:26:39,082 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-5) SQL Error: 0, SQLState: null
2020-05-29 11:26:39,082 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-5) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:26:39,086 INFO  [com.nec.asia.nic.framework.admin.code.dao.PaymentDefDaoTestCase] (pool-1-thread-5) start PaymentDef dao - update
2020-05-29 11:27:00,099 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-7) SQL Error: 0, SQLState: null
2020-05-29 11:27:00,100 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-7) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:27:00,101 ERROR [com.nec.asia.nic.framework.admin.code.service.CodesServiceTestCase] (pool-1-thread-7) error in Save: Could not open Hibernate Session for transaction; nested exception is org.hibernate.exception.GenericJDBCException: Could not open connection
2020-05-29 11:27:00,102 INFO  [com.nec.asia.nic.framework.admin.code.service.CodesServiceTestCase] (pool-1-thread-7) start CodesService - Update
2020-05-29 11:27:00,103 INFO  [com.nec.asia.nic.framework.admin.code.service.CodesServiceTestCase] (pool-1-thread-7)  end  CodesService - Update
2020-05-29 11:27:00,105 INFO  [com.nec.asia.nic.framework.admin.code.service.CodesServiceTestCase] (pool-1-thread-7) start CodesService - FindById
2020-05-29 11:27:21,103 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-8) SQL Error: 0, SQLState: null
2020-05-29 11:27:21,103 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-8) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:27:21,105 INFO  [com.nec.asia.nic.framework.admin.code.service.ParametersServiceTestCase] (pool-1-thread-8) start ParametersService - Update
2020-05-29 11:27:42,107 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-2) SQL Error: 0, SQLState: null
2020-05-29 11:27:42,108 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-2) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:27:42,111 INFO  [com.nec.asia.nic.framework.admin.code.dao.CodeDaoTestCase] (pool-1-thread-2) start Codes dao - find
2020-05-29 11:28:03,113 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-8) SQL Error: 0, SQLState: null
2020-05-29 11:28:03,114 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-8) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:28:03,118 INFO  [com.nec.asia.nic.framework.admin.code.service.ParametersServiceTestCase] (pool-1-thread-8) start ParametersService - FindById
2020-05-29 11:28:24,131 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-7) SQL Error: 0, SQLState: null
2020-05-29 11:28:24,132 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-7) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:28:24,134 INFO  [com.nec.asia.nic.framework.admin.code.service.CodesServiceTestCase] (pool-1-thread-7) start CodesService - findAll
2020-05-29 11:28:45,134 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-5) SQL Error: 0, SQLState: null
2020-05-29 11:28:45,135 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-5) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:28:45,137 INFO  [com.nec.asia.nic.framework.admin.code.dao.PaymentDefDaoTestCase] (pool-1-thread-5) start PaymentDef dao - find
2020-05-29 11:29:06,150 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-3) SQL Error: 0, SQLState: null
2020-05-29 11:29:06,152 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-3) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:29:06,160 INFO  [com.nec.asia.nic.framework.admin.code.dao.CodeValueDaoTestCase] (pool-1-thread-3) start CodeValue dao - save
2020-05-29 11:29:27,164 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-4) SQL Error: 0, SQLState: null
2020-05-29 11:29:27,168 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-4) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:29:27,175 INFO  [com.nec.asia.nic.framework.admin.code.dao.ParametersDaoTestCase] (pool-1-thread-4) start Parameters dao - find
2020-05-29 11:29:48,192 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-6) SQL Error: 0, SQLState: null
2020-05-29 11:29:48,193 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-6) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:29:48,197 INFO  [com.nec.asia.nic.framework.admin.code.dao.SiteRepositoryDaoTestCase] (pool-1-thread-6) start SiteRepository dao - find
2020-05-29 11:30:09,196 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-1) SQL Error: 0, SQLState: null
2020-05-29 11:30:09,197 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (pool-1-thread-1) Cannot create PoolableConnectionFactory (IO Error: The Network Adapter could not establish the connection)
2020-05-29 11:30:09,200 INFO  [com.nec.asia.nic.framework.admin.rbac.dao.FunctionsDaoTestCase] (pool-1-thread-1) start Functions dao - find
